integration/localnet/builder/bootstrap.go

```diff
@@ -458,6 +458,7 @@ func prepareObserverService(i int, observerName string, agPublicKey string) Serv
 		fmt.Sprintf("--bootstrap-node-public-keys=%s", agPublicKey),
 		fmt.Sprintf("--upstream-node-addresses=%s:%s", testnet.PrimaryAN, testnet.GRPCSecurePort),
 		fmt.Sprintf("--upstream-node-public-keys=%s", agPublicKey),
+		"--execution-data-indexing-enabled=true",
```

Comment:
we should also specify the db location
```suggestion
		"--execution-data-indexing-enabled=true",
		"--execution-state-dir=/data/execution-state",
```